svgnative/CMakeLists.txt

```diff
@@ -297,9 +348,46 @@ endif()
 ##############################
 # Creating pkg-config file
 ##############################
+include(GNUInstallDirs)
```

Comment:
Referring to the question about PkgConfig only being used with Cairo Graphics, shouldn't this be in a if clause for Cairo graphics as well?

Reply:
Sorry for making you confused. Even if pkg-config command is unavailable, we can generate pc files. To obtain the infos to be written in pc files, "make install" target should be prepared, so, GNUInstallDirs.cmake module is useful, regardless with Cairo availability.